distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -433,7 +434,8 @@ private static Integer findPortNumber(Integer portToTry) {
                 // increase timeout for CI environment
                 systemTimeout(TimeUnit.MINUTES.toMillis(10)),
                 when(Boolean.getBoolean("keepRuntimeFolder")).useOptions(keepRuntimeFolder()),
-                cleanCaches(true));
+                cleanCaches(true),
+                configureConsole().ignoreLocalConsole());
```

Comment:
What exactly is this doing to the console and why is it needed? I couldn't find good documentation on it.